uikit/src/main/java/com/midtrans/sdk/uikit/views/banktransfer/status/BankTransferStatusPresenter.java

```diff
@@ -67,8 +77,15 @@ public String getVaExpiration() {
                     expiration = transactionResponse.getPermataExpiration();
                     break;
                 case PaymentType.ALL_VA:
-                    //expiration is based on other VA processor
-                    expiration = TextUtils.isEmpty(transactionResponse.getBniExpiration()) ? transactionResponse.getPermataExpiration() : transactionResponse.getBniExpiration();
```

Comment:
have you tested this code miss behave or not working?

Reply:
Yes, I've tested that code. That code cannot handle bri as other VA Processor, that's why I changed it.